Description: Newer (pxe|sys)linux's localboot value produces unreliable 
 results when using documented options, -1 seems to provide the best
 supported value.
Author: Dave Walker (Daviey) <DaveWalker@ubuntu.com>
Forwarded: no
--- a/templates/pxe/pxedefault.template
+++ b/templates/pxe/pxedefault.template
@@ -8,7 +8,7 @@
 LABEL local
         MENU LABEL (local)
         MENU DEFAULT
-        LOCALBOOT 0
+        LOCALBOOT -1
 
 $pxe_menu_items
 
--- a/templates/pxe/pxelocal.template
+++ b/templates/pxe/pxelocal.template
@@ -5,5 +5,5 @@
 ONTIMEOUT local
 
 LABEL local
-        LOCALBOOT 0
+        LOCALBOOT -1
 
